index.js

```diff
@@ -370,7 +403,8 @@ class CoverageSonar extends CoverageBase {
 
         // Only get coverage percentage if the steps are finished
         if (projectKey && startTime && endTime) {
-            return getMetrics({ projectKey, startTime, endTime, prNum, sonarEnterprise })
+            return configureGitApp(projectKey, projectName)
```

Comment:
Configure git app will fire for each build?

Reply:
this shouldn't be put here. this will now happen for all UI calls to get metrics.

Reply:
Moved to create project flow